pandas/tests/arithmetic/test_numeric.py

```diff
@@ -662,7 +710,9 @@ def test_modulo2(self):
             result2 = p["second"] % p["first"]
             assert not result.equals(result2)
 
-            # GH#9144
+    def test_modulo_zero_int(self):
+        # GH#9144
+        with np.errstate(all="ignore"):
```

Comment:
RuntimeWarning?

Reply:
This is broken off of the previous test, which uses the context in the same way.

Reply:
k